glyphs-reader/src/glyphdata.rs

```diff
@@ -586,9 +586,9 @@ impl GlyphData {
         if let Some(base_names) = self.split_ligature_glyph_name(base_name) {
             let base_names_attributes: Vec<_> = base_names
                 .iter()
-                .filter_map(|name| self.query_no_synthesis(name, None))
+                .map(|name| self.query_no_synthesis(name, None))
                 .collect();
-            if let Some(first_attr) = base_names_attributes.first() {
+            if let Some(first_attr) = base_names_attributes.first().unwrap() {
```

Comment:
is this unwrap safe? base_names_attributes can't ever be empty because split_ligature_glyph_name always returns something? maybe write a comment